activities.yml

```diff
@@ -1,2989 +1,2479 @@
 <link>'s imagesrcset and imagesizes attributes:
   bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1515760
-  caniuse: null
-  description: Adds imagesrcset and imagesizes attributes to &lt;link> which correspond
-    to the srcset and sizes attributes of &lt;img> respectively, for the purposes
-    of preloading.
+  caniuse:
```

Comment:
We've a bunch of these empty values.  I assume that these can be dropped from the file safely.  Should they be?